LayoutTests/platform/mac/tables/mozilla/bugs/bug2886-2-expected.txt

```diff
@@ -3,105 +3,105 @@ layer at (0,0) size 800x600
 layer at (0,0) size 800x270
   RenderBlock {HTML} at (0,0) size 800x270
     RenderBody {BODY} at (88,25) size 648x220 [bgcolor=#FFFFF5]
-      RenderTable {TABLE} at (0,0) size 364x83
```

Comment:
Why are there changes here? Does this test use the new property?

Reply:
We've talked about this one offline. But just for registering it, it is styled by https://github.com/WebKit/WebKit/blob/main/LayoutTests/tables/mozilla/bugs/bug2886-2.css which uses font-size-adjust.